src/scripts/main.js

```diff
@@ -1,3 +1,204 @@
 'use strict';
 
-// write code here
+function getNumber(str) {
+  return str.includes('$')
+    ? +str.slice(1).replaceAll(',', '')
+    : +str;
+}
+
+function replaceFirstLetter(str) {
+  return str.slice(0, 1).toUpperCase() + str.slice(1);
+}
+
+function pushNotification(type, text, titleText) {
+  const boxNotification = document.createElement('div');
+  const h1 = document.createElement('h1');
+  const p = document.createElement('p');
+
+  boxNotification.append(h1, p);
+  boxNotification.classList.add('notification', type);
+  h1.className = 'title';
+  h1.innerText = titleText;
+  p.innerText = text;
+  document.body.append(boxNotification);
+
+  setTimeout(() => {
+    boxNotification.remove();
+  }, 3000);
+}
+
+function getThousandSeparator(number) {
+  return number.toLocaleString('en-Us');
+}
+
+const data = ['name', 'position', 'office', 'age', 'salary'];
+const offices = ['Tokyo',
+  'Singapore', 'London', 'New York', 'Edinburgh', 'San Francisco'];
```

Comment:
keep your code clean
```suggestion
const offices = [
    "Tokyo",
    "Singapore",
    "London",
    "New York",
    "Edinburgh",
    "San Francisco",
  ];
```